Modules/Microsoft365DSC/DSCResources/MSFT_PPTenantIsolationSettings/MSFT_PPTenantIsolationSettings.schema.mof

```diff
@@ -0,0 +1,17 @@
+[ClassVersion("1.0.0")]
+class MSFT_TenantRule
```

Comment:
Should we give the CIM Instance a more specific name (e.g. PPTenantRule) in case there are other workloads somewhere that also need to define tenantrules?